### PR TITLE
TODO STEF resolution pass

### PR DESCRIPTION
**Pull request by smagdali**
I've done another pass on TODO STEF comments. I've resolved some, and marked some as NOBLOCK if I don't think they're blockers to going live, and some as BOB where I think we could resolve quickly if @bobdubois and I worked discussed them live together, maybe on Monday morning. 

Some I've asked questions on slack to get answers I can plug in later. 

Also some more linty fixes.

### CODE REVIEW

**Review by ElPaisano — CHANGES_REQUESTED** [review]
Hey Stef, did a first pass through all of this. If it helps, I can do another pass with some linters I use once you and Bob have added in what you'd like to, Vale for spelling / duplicate words and markdown-link-check for external links. 

If you have any questions or comments about any suggestions made (or if you disagree or my suggestion is incorrect), please lmk and I am happy to discuss further. Also, I went through this pretty quick, so some of my own comments may be confusingly worded or formatted 😅 lmk

In this review, instead of directly making changes, I just made suggestions. However, I do think that the majority of these should be implemented before this is published. If it helps the team, I'm happy to implement all of these directly or, if the team would like to decide what they want, just implement the agreed upon changes. Or, the team can make the changes themselves - whatever works for y'all. Main thing is that I don't want to be a blocker to getting this published, so lmk what helps the most

A few general callouts:

1. Capitalization and formatting of terms: there's a general inconsistency throughout the PR in how terms (i.e. Storage Providers) are capitalized (fully, partially, or not at all) and formatted (quotes, italics, not at all, etc.). We should pick one or define a consistent rule.
2. Introduction of terms: in general, introduce the full term first i.e. Committed Capacity, followed by the abbreviation in parens i.e. (CC). I've noticed this also was inconsistent throughout the branch https://docs.ipfs.tech/community/contribute/grammar-formatting-and-style/#grammar-and-spelling
3. Headers. This is based off of the [ipfs-docs style guide](https://docs.ipfs.tech/community/contribute/grammar-formatting-and-style/#titles) (which I believe was the same as what we had in filecoin-docs at one point):

    - header capitalization is inconsistent. Unless it's a term like "Big Data Exchange", only the first word should be capitalized
   - remove special characters like "&"
   - remove punctuation like "?", rewrite

4. Use bold text for buttons, hyperlinks, images with text in them, window names, and icons. https://docs.ipfs.tech/community/contribute/grammar-formatting-and-style/#style

https://docs.ipfs.tech/community/contribute/grammar-formatting-and-style/#grammar-and-spelling

I'm cc'ing @johnnymatthews on these to sanity check me here (johnny please lmk if I'm incorrect here). Wanted to point to the docs grammar / styling guide as the source of truth, but it appears to have been removed? https://docs.filecoin.io/basics/project-and-community/ways-to-contribute/#writing-guide. I generally based my review off of https://docs.ipfs.tech/community/contribute/grammar-formatting-and-style/#grammar-and-spelling, which I seem to recall was basically the same style guide as Fil docs

File: content/en/storage-provider/architecture/Lotus-components/index.md [hunk 1]
```diff
@@ -104,27 +106,27 @@ The latter file, which lives under your storage path,is also required for restor
...
 For information on how to check if there are upcoming proving deadlines, see [Helpful commands]({{< relref "#view-proving-deadlines" >}}).
 {{< /alert >}}
 
 ## Lotus worker
 
-The Lotus worker is the 3rd important software component in the Lotus architecture. There can be - and most likely will be - multiple workers in a single Storage Provider setup. Assigning designated roles to each worker in the setup allows for scaling the setup for higher throughput, [sealing Rate]({{<relref "sealing-rate">}})) and improved redundancy.
+The Lotus worker is the 3rd important software component in the Lotus architecture. There can be - and most likely will be - multiple workers in a single Storage Provider setup. Assigning designated roles to each worker in the enables higher throughput, [sealing rate]({{<relref "sealing-rate">}}), and improved redundancy.
 
 As mentioned above, proving tasks can be assigned to dedicated workers, and workers can also get storage access.
 The remaining worker tasks are running [Sealing Pipeline]({{<relref "sealing-pipeline">}}) which is discussed in the next section.
```
[the remark this answers is not in the document]
Comment (smagdali) [reply content/en/storage-provider/architecture/Lotus-components/index.md:132]: also fixed the Sealing Pipeline incorrect captilisation

File: content/en/storage-provider/filecoin-deals/verified-deals/index.md [hunk 2]
```diff
@@ -14,31 +14,39 @@ menu:
...
+
+With great power comes great responsibility, which also counts for storage power: rewards on Filecoin Plus deals are 10x, but so are the penalties. Because a sector of 32GiB counts for 320GiB of storage power (10x), the rewards and the penalties are calculated on the QAP of 320GiB. Filecoin Plus allow a Storage Provider to earn more block rewards on a verified deal, compared to a regular data deal. The 10x multiplier on storage power that comes with a verified deal, however, also requires 10x collateral from the Storage Provider.
+
+If the Storage Provider is then not capable of keeping the data and systems online and fails to submit the daily required proofs (WindowPoSt) for that data, the penalties ('slashing') are also 10x higher than over regular data deals or CC sectors. Larger storage power means larger block rewards, larger collateral and larger slashing. The stakes are high - after all, we’re “storing humanity’s most important information” with Filecoin.
```
[the remark this answers is not in the document]
Comment (smagdali) [reply content/en/storage-provider/filecoin-deals/verified-deals/index.md:52]: Yup, I'm not a fan of alleged "quotes" either :)

**Comment by ElPaisano** [comment]
Howdy @smagdali remaining edits made in https://github.com/filecoin-project/filecoin-docs/pull/1883/commits/287c27d1df0907794b4248ebd2701a7774d0c540 🚀 

Note that I found a few more (super nitpicky) things, which I also corrected in here.

Please lmk if there are any concerns here or if you need anything else on this one. Thank you

**Review by smagdali — COMMENTED** [review]
LGTM

**Pull request merged**